assignment.js

```diff
@@ -113,6 +147,42 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+let playerAChoice = "rock"
+let playerBChoice = "paper"
+
+if (playerAChoice === playerBChoice) {
+  console.log("Result: Tie")
+}
+
+else if (playerAChoice === "rock"){
+  if (playerBChoice === "paper")  {
+  console.log("Result: PlayerB wins")
+  }
+  else if (playerBChoice === "scissor"){
+  console.log("Result: PlayerA wins")
+  }
+}
+
+
+else if (playerAChoice === "scissors"){
+  if (playerBChoice === "rock")  {
+  console.log("Result: PlayerB wins")
+  }
+  else if (playerBChoice === "paper"){
+  console.log("Result: PlayerA wins")
+  }
+}
+
+else if (playerAChoice === "paper"){
+  if (playerBChoice === "scissors")  {
+  console.log("Result: PlayerB wins")
+  }
+  else if (playerBChoice === "rock"){
+  console.log("Result: PlayerA wins")
+  }
+}
+
```

Comment:
@AtleLohrmann Good work here with creating your game. The conditionals are vital for this solution and you implemented them very well checking to see who wins the game. One suggestion, I would recommend adding one more conditional statement that would return an error if the player choice doesn't match any of the valid options.
Take a look at this approach [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L129)